src/model/guild/member.rs

```diff
@@ -508,7 +513,7 @@ impl Member {
     /// [Ban Members]: Permissions::BAN_MEMBERS
     #[inline]
     pub async fn unban(&self, http: impl AsRef<Http>) -> Result<()> {
```

Comment:
Why don't you add support to the reason message here (and in others structures above)? Is it in WIP?

Reply:
This PR only adds audit log support to the Http client for simplicity. I will expand the explanation for this in the PR description, hopefully that will clear it up